RebelAllianceBank/Classes/Customer.cs

```diff
@@ -71,33 +72,107 @@ public void CreateAccount()
                     accountCurrency = Console.ReadLine().ToUpper();
                 }
 
-                switch (userChoice)
+                //switch (userChoice)
+                //{
+                //    case 1:
+                //        BankAccounts.Add(new CardAccount(accountName, 0, accountCurrency, 0.0m));
+                //        createAccount = true;
+                //        Console.ReadKey();
+                //        break;
+                //    case 2:
+                //        BankAccounts.Add(new ISK(accountName, 0, accountCurrency, 0.0m));
+                //        createAccount = true;
+                //        Console.ReadKey();
+                //        break;
+                //    case 3:
+                //        BankAccounts.Add(new SavingsAccount(accountName, 0, accountCurrency, 0.0m));
+                //        createAccount = true;
+                //        break;
+                //    case 4:
+                //        createAccount = true;
+                //        break;
+                //    default:
+                //        Console.WriteLine("Fel inmatning, inget konto har skapats.");
+                //        Console.ReadKey();
+                //        Console.Clear();
+                //        createAccount = false;
+                //        break;
+                //}
+            } while (createAccount == false);
+        }
+        /// <summary>
+        /// A method to transfer from one of current users account to another users account.
+        /// </summary>
+        /// <param name="currentUser">Input personal number of logged in user</param>
+        /// <param name="users">A list of all users</param>
+        public void TransferUserToUser(string currentUser, List<IUser> users)
+        {
+            Customer thecurrentUser = users.OfType<Customer>().First(user => user.PersonalNum == currentUser);
```

Comment:
Why do you take in a string called currentUser. Wouldn't a currentUser of the IUser type be a better option. That way you don't have to have the long lamda expression to set thecurrentUser variable.

Reply:
Changed the parameter of the method to call for Customer currentUtser instead of string currentUser. Resulting in removal of code on line 110.